charts/crunchy-postgres/templates/PostgresCluster.yaml

```diff
@@ -102,47 +101,72 @@ spec:
   backups:
     pgbackrest:
       {{ if .Values.pgBackRest.image }}
-      image: {{ .Values.pgBackRest.image }}
+      image: {{ .Values.pgBackRest.image | quote }}
       {{ end }}
-      {{- if .Values.pgBackRest.s3.enabled }}
+      {{ if eq $.Values.pgBackRest.s3.enabled true }}
       configuration:
       - secret:
-          name: {{ .Values.pgBackRest.s3.s3Secret }}
-      {{- end }}
+          name: {{ .Values.pgBackRest.s3.s3Secret | quote }}
+      {{ end }}
       global:
-        # Support both PVC and s3 backups
-        repo1-retention-full: {{ .Values.pgBackRest.retention | quote }}
-        repo1-retention-full-type: {{ .Values.pgBackRest.retentionFullType }}
-        {{- if .Values.pgBackRest.s3.enabled }}
-        repo2-retention-full: {{ .Values.pgBackRest.retention | quote }}
-        repo2-retention-full-type: {{ .Values.pgBackRest.retentionFullType }}
-        repo2-path: {{ .Values.pgBackRest.s3.s3Path }}
-        repo2-s3-uri-style: {{ .Values.pgBackRest.s3.s3UriStyle }}
-        {{- end }}
+      {{ range $repo := .Values.pgBackRest.repos }}
+        {{ if eq $.Values.pgBackRest.s3.enabled true }}
```

Comment:
This could probably be changed to check if the s3 key exists, using this check for the bucket path, also did not want to change the other uses of s3.enabled